cardazim/server.py

```diff
@@ -0,0 +1,51 @@
+import socket
+import argparse
+import sys
+
+def listener_server(server_ip: str, server_port: int):
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as server_socket:
+        server_socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        server_socket.bind((server_ip, server_port))
+        server_socket.listen(0)
+
+        while True:
+            client_socket, client_addr = server_socket.accept()
+
+            with client_socket:
+                from_client = ''
+
+                while True:
+                    data = client_socket.recv(4096)
+                    if not data:
+                        break
+
+                    from_client += data.decode('utf8')
```

Comment:
this logic should be in a different function. good general rule for python (when you will get to react it would not be that easy :) ) is the 5 lines rule. this means every function should be no longer then five lines. this rule is not mandatory and in fact is not always good to follow that rule but it will make you thing "is this function too long? where can i split it?"